classes/class-dintero-checkout-gateway.php

```diff
@@ -101,7 +112,25 @@ public function get_icon() {
 		 * @return boolean
 		 */
 		public function is_available() {
-			return ( 'yes' === $this->enabled );
+			if ( ! 'yes' === $this->enabled ) {
```

Comment:
Change this to the proper way of doing a negative check. This adds a unnecessary extra check by first matching the value and then checking if its true/false
`( 'yes' !== $this->enabled )`